### DISCUSSION TITLE
Rectangle support for Alfred workflow

### DISCUSSION THREAD

**Discussion by sudarshang** [rxhanson/Rectangle#584] [Q&A] [answered]
Is there a way to script Rectangle - what I am looking for is a way to have a shortcut that not only activates a window (like Cmd + Tab) but also positions it using Rectangle?

**Comment by rxhanson** [comment]
There is no scripting at this time, but I am open to a pull request for it.

**Reply by Zoomer547547** [reply]
This would be extremely handy. I use 2x 40" monitors and setting custom locations for side by side work windows that open when I run an Alfred5 workflow would be awesome.

**Reply by rxhanson** [reply]
(A while back, scripting was added in. It's done by URL: https://github.com/rxhanson/Rectangle#execute-an-action-by-url)

**Comment by notDavid** [comment]
Fyi for people stumbling upon this: https://github.com/gohoyer/Alfred-Rectangle-Shortcuts

**Comment by rxhanson** [comment, marked as answer]
A while back, scripting was added in. It's done by URL: https://github.com/rxhanson/Rectangle#execute-an-action-by-url
